testdata/project-v2/Makefile

```diff
@@ -52,7 +52,7 @@ vet:
 
 # Generate code
 generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+	go run sigs.k8s.io/controller-tools/cmd/controller-gen object:headerFile=./hack/boilerplate.go.txt paths="./..."
```

Comment:
Did you do this change manually or did you run the generate golden script?

Reply:
that was manually, here https://github.com/kubernetes-sigs/kubebuilder/commit/e75141929f0f6e626b5087fc4b675ddcc6115148, i can revert that if you want

Reply:
No, what I mean is that `object:headerFile=./hack/boilerplate.go.txt` should be `object:headerFile="hack/boilerplate.go.txt"` (#1288)

Reply:
Can someone explain to me why in the tests this line leads to this diff (and thus the [test fails](https://travis-ci.org/kubernetes-sigs/kubebuilder/builds/634870273?utm_source=github_status&utm_medium=notification))

```diff
diff --git a/testdata/project-v2/Makefile b/testdata/project-v2/Makefile
index 9992458..5e20236 100644
--- a/testdata/project-v2/Makefile
+++ b/testdata/project-v2/Makefile
@@ -52,7 +52,7 @@ vet:
 
 # Generate code
 generate: controller-gen
-	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths="./..."
+	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 # Build the docker image
 docker-build: test
```

Reply:
It is a bug introduced by other commit that got merged. #1288 should fix it, but you can also change that if you want.

Reply:
@Adirio perfect, I merged your pull request https://github.com/kubernetes-sigs/kubebuilder/pull/1289/commits/af9bb4ea4cf67a9fcdd09d7bd1b9f55d859cfd87

This branch now got pretty big... maybe I should squash everything?

Reply:
Yeah go for it, squash and force-push